source/common/upstream/cluster_manager_impl.cc

```diff
@@ -841,8 +841,13 @@ ThreadLocalCluster* ClusterManagerImpl::get(absl::string_view cluster) {
 }
 
 void ClusterManagerImpl::maybePrefetch(
-    ThreadLocalClusterManagerImpl::ClusterEntryPtr& cluster_entry,
+    ThreadLocalClusterManagerImpl::ClusterEntryPtr& cluster_entry, const ClusterConnectivityState&,
```

Comment:
It might be better to split this part of the change out from the accounting work but up to you.